packages/brokeneck-react/src/components/Groups.js

```diff
@@ -24,19 +24,69 @@ import usePagination from '../hooks/usePagination'
 import { LOAD_GROUPS } from '../graphql'
 import useProvider from '../hooks/useProvider'
 import useSearch from '../hooks/useSearch'
+import PlusIcon from '../icons/plus'
+import RefreshIcon from '../icons/refresh'
 
 import Square from './Square'
 
 const useStyles = makeStyles(theme => ({
+  actions: {
+    alignItems: 'center',
+    borderLeft: `1px solid ${theme.palette.separator.main}`,
+    display: 'flex',
+    marginLeft: theme.spacing(2.5),
+    marginTop: 10,
+    paddingLeft: theme.spacing(1)
+  },
+  actionButton: {
+    textTransform: 'none',
+    '& svg': {
+      fill: theme.palette.primary.main,
+      marginLeft: theme.spacing(1)
+    }
+  },
+  header: {
+    alignItems: 'center',
+    display: 'flex',
+    marginLeft: theme.spacing(2.5)
+  },
+  right: {
+    marginLeft: 'auto'
+  },
+  search: {
+    marginLeft: 'auto'
+  },
+  separator: {
+    borderLeft: `1px solid ${theme.palette.separator.main}`,
+    height: theme.spacing(2),
+    marginLeft: theme.spacing(1),
+    marginRight: theme.spacing(1)
+  },
   spacing: {
     display: 'flex',
     alignItems: 'center',
     '& > * + *': {
       marginLeft: theme.spacing(2)
     }
   },
-  right: {
-    marginLeft: 'auto'
+  tableHead: {
+    '& th': {
+      backgroundColor: theme.palette.headerBackground.main,
+      borderBottom: 0,
+      fontWeight: 'bold'
+    },
+    '& th:first-child': {
+      borderRadius: '10px 0 0 10px'
+    },
+    '& th:last-child': {
+      borderRadius: '0 10px 10px 0'
+    }
+  },
+  tableRow: {
+    cursor: 'pointer',
+    '&:hover': {
+      backgroundColor: theme.palette.tableRowHighlight.main
+    }
```

Comment:
if we want a common table styles, this should probably live elsewhere and not in the component. mui has global overrides

Reply:
Good point, I'll look at the global overrides configuration in the theme

Reply:
this page has all the customization options for the different scenarios. https://material-ui.com/customization/components. this section is probably what you're looking for here https://material-ui.com/customization/components/#global-theme-override